.github/workflows/altdoc.yaml

```diff
@@ -34,15 +34,18 @@ jobs:
 
       - name: Dependencies
         run: ./run.sh install_all
+        with:
+          needs: website
+          cache: "always"
```

Comment:
You need the logical equivalent of the declaration of line 55 below added here, potentially in two (OS-dependent) blocks depending on which OS you want to run here.  If it is 'merely' an Linux-only documentation build step you do not need to condition and can just list what is needed as arguments to `apt`, or to `install.r` (wrapping `install.packages()`) if you want it to be R.  Similarly, `installGithub.r` fetches from where you think it does etc.